lib/srv/desktop/windows_server_test.go

```diff
@@ -16,9 +16,9 @@ package desktop
 
 import (
 	"context"
+	"crypto/rand"
```

Comment:
I may have missed this in the original PR, but I'm not sure why this is necessary. `math/rand` should be totally fine here. Just generating dummy data, not using it for cryptographic purposes.

Reply:
math/rand is essentially deprecated in favor of crypto/rand: https://pkg.go.dev/math/rand#Read. (`staticcheck` catches it - https://staticcheck.io/docs/checks#SA1019)

Reply:
TIL, thanks!